From 298bc488fdc047da37482f4003023cb9adef78f8 Mon Sep 17 00:00:00 2001
From: Florian Weimer <fweimer@redhat.com>
Date: Wed, 11 Sep 2024 10:05:08 +0200
Subject: [PATCH] manual: Extract the @manpageurl{func,sec} macro
Content-type: text/plain; charset=UTF-8

From the existing @manpagefunctionstub{func,sec} macro,
so that URLs can be included in the manual without the
stub text.

Reviewed-by: Carlos O'Donell <carlos@redhat.com>
---
 manual/macros.texi | 7 +++++--
 1 file changed, 5 insertions(+), 2 deletions(-)

diff --git a/manual/macros.texi b/manual/macros.texi
index 579da3fb81..f48dd4ec22 100644
--- a/manual/macros.texi
+++ b/manual/macros.texi
@@ -282,10 +282,13 @@ cwd\comments\
 @macro standardsx {element, standard, header}
 @end macro
 
+@macro manpageurl {func, sec}
+@url{https://man7.org/linux/man-pages/man\sec\/\func\.\sec\.html}
+@end macro
+
 @macro manpagefunctionstub {func,sec}
 This documentation is a stub.  For additional information on this
-function, consult the manual page
-@url{https://man7.org/linux/man-pages/man\sec\/\func\.\sec\.html}.
+function, consult the manual page @manpageurl{\func\,\sec\}.
 @xref{Linux Kernel}.
 @end macro
 
-- 
2.43.5

